Xamarin.Forms.Platform.iOS/Renderers/PageRenderer.cs

```diff
@@ -80,7 +80,7 @@ public override void ViewDidAppear(bool animated)
 			UpdateStatusBarPrefersHidden();
 		}
 
-		public override void ViewDidDisappear(bool animated)
+		public override async void ViewDidDisappear(bool animated)
```

Comment:
making this `async void` will be `fire and forget`, and the thread calling this will continue wether or not this method is completed. That's probably _not_ what we want.